src/main/java/edu/cornell/kfs/fp/batch/xml/DisbursementVoucherDetailXml.java

```diff
@@ -0,0 +1,109 @@
+package edu.cornell.kfs.fp.batch.xml;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.adapters.NormalizedStringAdapter;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import org.apache.commons.lang3.StringUtils;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "DisbursementVoucherDetail", namespace = StringUtils.EMPTY)
```

Comment:
Does this correspond to dv_detail on the documentation?

Reply:
Yes, and the dv_detail is in the annotation on AccountingXmlDocumentEntry